.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,51 @@
+name: Feature Request
+description: Suggest a new feature or enhancement
+labels: [ "enhancement | feature :zap:" ]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please consider whether your changes are useful for all users, or if this is
+        specific to your use case.
```

```suggestion
        Please consider whether your changes are useful for all users, or if this is specific to your use case.
```

The line break is a bit weird^^